model/patch/patch.go

```diff
@@ -301,6 +302,18 @@ func (p *Patch) SetParameters(parameters []Parameter) error {
 	)
 }
 
+func (p *Patch) SetDownstreamParameters(parameters []Parameter) error {
+	p.Triggers.DownstreamParameters = parameters
+	return UpdateOne(
+		bson.M{IdKey: p.Id},
+		bson.M{
+			"$set": bson.M{
+				TriggerInfoDownstreamParametersKey: parameters,
```

Comment:
I think this needs to be something like:
```
bsonutil.GetDottedKeyName(TriggersKey, TriggerInfoDownstreamParametersKey): parameters,
```

This corresponds to an update like this in the mongo shell:
```
db.patches.updateOne({_id:"the_id"}, {"$set": {"triggers.downstream_parameters": [the parameters]}})
```